web/components/login/Login.js

```diff
@@ -26,11 +26,15 @@ class Login extends React.Component {
 
     this.login = (event) => {
       event.preventDefault();
-      this.props.firebase.login({
+      const credential = {
         email: this.state.email,
         password: this.state.password,
-      }).then(() => {
+      };
+
+      this.props.firebase.login(credential).then(() => {
         this.setState({ loggedIn: true });
+      }).catch((error) => {
+        console.log(error);
```

Comment:
might want to do something like `this.setState({ errored: true})` for error message display instead of failing silently, though I'm not sure if we have plans for more global error handling

Reply:
Yeah i definitely agree! I created a separate issue for that here (https://www.notion.so/nwplus/Handle-Login-Error-6e3d864cca474b8bab4d21f936699fe6), didn't want to make this into a huge PR